src/ProxyQuery/Doctrine/ProxyQuery.php

```diff
@@ -27,7 +27,7 @@ public function execute(array $params = [], ?int $hydrationMode = null)
         $sortBy = $this->getSortBy();
         $sortOrder = $this->getSortOrder();
 
-        if (null !== $sortBy && null !== $sortOrder) {
+        if ([] !== $sortBy && [] !== $sortOrder) {
```

Comment:
This looks still wrong, because line 33 tries to concat a string value

Reply:
Yes, I used null because of that, the methods phpdoc says mixed, the property is initialized to array, but here is used as string on the sprintf